components/extractors/marketing-sales/shoptet/index.md

```diff
@@ -8,7 +8,13 @@ redirect_from:
 * TOC
 {:toc}
   
-This extractor allows you to download data from Shoptet permalinks.
+This extractor allows you to download data from Shoptet permalinks. 
+In order to be able to use this extractor, you have to create a permanent link to your report in Shoptet administration.
+
+For example, to create a permanent link for Orders, go to Orders > Export in you Shoptet adminitration page and at the bottom of the page, you can see the option to create the permanent link. This link is a direct download link to the csv file that you will use in component's configuration. To see how the report looks like, you can visit this link using your browser and it will automatically get downloaded to your computer.
```

Comment:
@SgtMarmite can you include mention and stress out that it needs to be CSV format and include screenshot from Shoptet + link their docs https://podpora.shoptet.cz/hc/cs/articles/360003161791-Export-produkt%C5%AF
![image](https://github.com/keboola/connection-docs/assets/15654252/8abacaf5-510e-4b21-a188-b75ff79c61d9)

It needs to be clear that anything else then CSV won't work